libbeat/idxmgmt/ilm/std.go

```diff
@@ -114,26 +114,26 @@ func (m *singlePolicyManager) EnsureAlias() error {
 	return m.client.CreateAlias(m.alias)
 }
 
-func (m *singlePolicyManager) EnsurePolicy(overwrite bool) error {
+func (m *singlePolicyManager) EnsurePolicy(overwrite bool) (bool, error) {
 	log := m.log
 	overwrite = overwrite || m.overwrite
 
 	exists := true
 	if m.checkExists && !overwrite {
 		b, err := m.client.HasILMPolicy(m.policy.Name)
```

Comment:
I know this is not part of this PR, sorry, but is var `b` needed at all?
we not using `exists` as the receiving param

Reply:
`err` doesn't exist yet.